jsk_pcl_ros/include/jsk_pcl_ros/pcl/extract_indices_patch.h

```diff
@@ -47,11 +47,34 @@ pcl::ExtractIndices<pcl::PCLPointCloud2>::applyFilter (PCLPointCloud2 &output)
   if (keep_organized_)
   {
     output = *input_;
-    for (size_t i = 0; i < indices_->size (); ++i) {
-      for (size_t j = 0; j < output.fields.size(); ++j) {
-        memcpy (&output.data[(*indices_)[i] * output.point_step + output.fields[j].offset],
-                &user_filter_value_, sizeof(float));
-      }
+    if (negative_)
+    {
+      // Prepare the output and copy the data
+      for (size_t i = 0; i < indices_->size (); ++i)
```

Comment:
I'd like to keep this file same as the one in pcl. Would you mind that?
In pcl library the above seems the common style.